src/main/java/pl/project13/jgit/JGitCommon.java

```diff
@@ -97,16 +80,16 @@
     }
   }
 
-  public String getClosestTagName(@NotNull LoggerBridge loggerBridge,@NotNull Repository repo){
-    Map<ObjectId, List<DatedRevTag>> map = getClosestTagAsMap(loggerBridge,repo);
+  public String getClosestTagName(@NotNull Repository repo, @NotNull Mojo mojo){
```

Comment:
We have an effort attempting to make the core of this lib not depend on maven, so gradle/sbt/lein can use it - let's keep the LoggerBridge please, even if it's only backed by the Mojo now,